src/content/developers/docs/spin-up-your-node/index.md

```diff
@@ -0,0 +1,146 @@
+Censorship resistant and decentralized network should not rely on cloud providers infrastructure. For running Ethereum node locally, check Hardware section below. Easiest options are preconfigured machines like [DappNode](https://dappnode.io/) or [Avado](https://ava.do/). 
+
+#### Hardware {#hardware}
+
+Minimum and recommended requirements are summed up in [Nodes and clients page section](/en/developers/docs/nodes-and-clients/#requirements). 
+Generally, modest computing power should be enough, bottleneck is usually speed of a drive. During initial sync, Ethereum client performs a lot of read/write operations, therefore SSD is strongly recommended. Client might even [not be able to sync current state on HDD](https://github.com/ethereum/go-ethereum/issues/16796#issuecomment-391649278) and get stuck few blocks behind mainnet. 
+Some clients offer builds for different architectures which makes possible to run it on [SCBs with ARM](/en/developers/docs/nodes-and-clients/#ethereum-on-a-single-board-computer/). Another easy way is using [Ethbian](https://ethbian.org/index.html) operating system for Raspberry Pi 4 which enables [run Ethereum client by simply flashing SD card](/en/developers/tutorials/run-node-raspberry-pi/).
+Depending on the chosen software and the hardware, the initial synchronization time and storage requirements may vary. Check [orientational table](/en/developers/docs/nodes-and-clients/#recommended-specifications) in Nodes and clients page. 
+Also make sure your internet connection is not limited by [bandwidth cap](https://en.wikipedia.org/wiki/Data_cap). It is recommended to use unmetered connection since initial sync and network gossip could exceed limits of your ISP. 
+
+#### Operating system {#operating-system}
+
+All clients support major operating systems - Linux, MacOS, Windows. This enables to run client on regular desktop or server machine with OS that suits user the best. Make sure OS is up to date to avoid potential issues and security vulnerabilities. 
+
+
+## Spinning up the node {#spinning-up-node}
+
+### Getting the client software {#getting-the-client}
+
+First, let's download chosen client software itself. 
+
+Users can simply download executable application or installation package which suits their operating system and architecture. It is always recommended to verify signatures and checksums of downloaded packages. Some clients also offer repositories for easier installation and updates. 
+Developers might prefer building from source. All of the clients are open source and it is easy to build them from source code with proper compiler. 
+
+Stable mainnet client implementations, summarized on page above, can be downloaded from their release pages - [Geth](https://geth.ethereum.org/downloads/), [OpenEthereum,](https://github.com/openethereum/openethereum/releases) [Nethermind](https://downloads.nethermind.io/), [Besu](https://pegasys.tech/solutions/hyperledger-besu/). 
```

Comment:
This paragraph not needed if we link to the client table.

Alternatively we could just bullet point them here, we don't need to reference "page above"